cmd/run.go

```diff
@@ -93,11 +93,15 @@ func bindRunFlags(cmd *cobra.Command, config *app.Config) {
 	cmd.Flags().StringVar(&config.TestnetConfig.CapellaHardFork, "testnet-capella-hard-fork", "", "Capella hard fork version of the custom test network.")
 	cmd.Flags().StringVar(&config.ProcDirectory, "proc-directory", "", "Directory to look into in order to detect other stack components running on the host.")
 	cmd.Flags().StringVar(&config.ConsensusProtocol, "consensus-protocol", "", "Preferred consensus protocol name for the node. Selected automatically when not specified.")
+	cmd.Flags().StringVar(&config.Nickname,"nickname","","Human readable peer nickname. Maximum 32 characters.")
```

Comment:
Let's say "human friendly" maybe instead of "readable".
Also, I see no spacing around commas, see other lines above.
I suspect your golang code formatting is not set properly for IDE.